.github/workflows/kfp-sdk-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: KFP SDK Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'sdk/**'
+      - 'test/presubmit-tests-sdk.sh'
+  pull_request:
+    paths:
+      - 'sdk/**'
+      - 'test/presubmit-tests-sdk.sh'
```

Comment:
We should also add `.github/workflows/kfp-sdk-tests.yml` here.

Reply:
Done, added.

Reply:
I don't see this change.

Reply:
Pushed again, changes should be visible now.